src/main/python/TravelTimeReporter.py

```diff
@@ -102,6 +102,29 @@ def read_skims(self, mode = "transit"):
             
         skims.close()
 
+        # Read bike and walk times from AM traffic skim
+        am_traffic_skim_file = os.path.join(
+            self.model_run,
+            "output",
+            "skims",
+            "traffic_skims_AM.omx"
+        )
+        skims = omx.open_file(am_traffic_skim_file, "r")
+        for core in ["BIKE_TIME", "walkTime"]:
+            skim_values = np.array(skims[core.format(self.settings["time_period"])])
+            skim_values = np.where(
+                skim_values == 0,
+                self.settings["infinity"],
+                skim_values
+            )
+            self.skims[core] = self.expand_skim(
```

Comment:
self.skims contains both "bike_time" and "BIKE_TIME" keys. Python dictionary is case sensitive so this should be fine

Reply:
Thanks for catching that. It may be worth it to remove one of them to make the script cleaner and more efficient.

Reply:
Yes please, no duplicates. What about walktime? Do we need to update the keys in the two yaml files? https://github.com/SANDAG/ABM/blob/ABM3_develop/src/main/python/AMTravelTimeReporterConfigs.yaml#L23

Reply:
I just realized that I forgot to have the script call the function that adds the TAZ-level skim times...